rosetta/transactions/parse_transaction.go

```diff
@@ -128,8 +128,9 @@ func (p *Parser) ParseTransaction(tx *flow.Transaction) ([]object.Operation, []i
 				failure.WithErr(err)),
 		}
 	}
+	addr := flow.HexToAddress(val.String())
```

Comment:
This chomps the leading `0x` from the address, as it's stored in the Cadence transaction argument.